Backend/src/main/java/com/onbank/api/model/Transfer.java

```diff
@@ -20,53 +19,62 @@
     @Id
     @Column(name = "id", nullable = false, unique = true)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @CsvBindByName
     private Long id;
 
     @Column(name = "date", nullable = false)
-    @NotNull(message = "Date cannot be empty.")
+    @CsvBindByName
     private LocalDate date;
 
     @Column(name = "recipientname", nullable = false, length = 200)
-    @NotNull(message = "Recipient cannot be empty.")
+    @CsvBindByName
     private String recipientName;
 
     @Column(name = "recipientaccountnumber", nullable = false)
     @Size(min = 26, max = 28, message = "Account number must be between 26-28 characters.")
     @NotNull(message = "Account number cannot be empty.")
     @Pattern(regexp = "(^[A-Z]{2}\\d{26}$)|(^\\d{26}$)")
     @AccountNumber(message = "Invalid account number.")
+    @CsvBindByName
     private String recipientAccountNumber;
 
     @Column(name = "description", length = 4000)
+    @CsvBindByName
     private String description;
 
     @NotNull(message = "Type of operation cannot be empty.")
     @Column(name = "typeofoperation", nullable = false, length = 200)
     @Enumerated(EnumType.STRING)
-    private OperationType OperationType;
+    @CsvBindByName
```

Comment:
Encja nie powinna wiedzieć o tym że jest wykorzystywana przez zewnętrzną biblioteke do parsowania csv, to jest pomieszanie wartstw aplikacji.

Reply:
Mój błąd, kompletnie o tym zapomniałem. Na refactorze miało to być usunięte.